plugins/processors/awsapplicationsignals/common/types.go

```diff
@@ -28,6 +28,14 @@ const (
 	AttributeHost                = "Host"
 )
 
+// Platform attribute used as CloudWatch EMF log field.
+const (
+	MetricAttributeECSCluster                = "ECS.Cluster"
+	MetricAttributeECSTaskId                 = "ECS.TaskId"
+	MetricAttributeECSTaskDefinitionFamily   = "ECS.TaskDefinitionFamily"
+	MetricAttributeECSTaskDefinitionRevision = "ECS.TaskDefinitionRevision"
```

Comment:
Is it intentional to have nothing about the ECS.Service?

Reply:
Yes, today, ECS.Service is not available in the otel attributes.